controls.py

```diff
@@ -0,0 +1,118 @@
+import pygame
+import serial
+import sympy
+import numpy
+import matplotlib
+import PySimpleGUI as sg
+import time
+class Control():
+    #Creates variables that store the motor labels for the different motors used. Some motor values have also been put in for adjustments that may be made to add the claw          
+    _CENTRAL_MOTOR = 1
+    _RIGHT_MOTOR = 2
+    _LEFT_MOTOR = 3
+    _UP_QUANTITY = 500
+    _RIGHT_QUANTITY = 500
+    _LEFT_QUANTITY = 500
+
+    def __init__(self, scale, adjustment, running, list_movements, controller, clock, serial_port, serial_baudrate):
+        self.scale = scale 
+        self.adjustment = adjustment
+        self.running = False
+        self.list_movements = {
+            abutton: 0,
+            bbutton: 0,
+            xbutton: 0,
+            ybutton: 0,
+            left_motor_right: 0,
+            up_motor : 0,
+            right_motor_right: 0,
+            turning_amount: 0
+        }
+        
+        #Initializes the joystick being used, which is the XBox Controller
+        
+        self.controller = pygame.joystick.Joystick(0)
+        #Sets up the timing for the interface
+        
+        self.clock = pygame.time.Clock()
+        
+        self.serial_port = "/dev/ttyUSB0"
```

Comment:
Should this be `self.serial_port = serial_port`? Same with with `self.serial_baudrate`.

Reply:
Yeah it should